lib/daru.rb

```diff
@@ -26,19 +26,20 @@ module Daru
     10 => 31,
     11 => 30,
     12 => 31
-  }
+  }.freeze
 
-  SPLIT_TOKEN = ','
-  class << self
-    @@lazy_update = false
+  @lazy_update = false
```

Comment:
There are [many](http://archive.oreilly.com/pub/a/ruby/excerpts/ruby-best-practices/worst-practices.html) [rants](http://4thmouse.com/index.php/2011/03/20/why-class-variables-in-ruby-are-a-bad-idea/) on them. Neither of the problems listed are applicable to our case, but I think class variables are considered code smell nowadays unconditionally.